docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+jupyter-book
+matplotlib
+numpy
```

Comment:
Please add these to the `pyproject.toml` instead as an optional 'docs' dependencies.

Reply:
Updated as per comment.